x/common/util.go

```diff
@@ -326,3 +326,25 @@ func ValidateUint16Positive(param string) subspace.ValueValidatorFn {
 		return nil
 	}
 }
+
+// CheckSignerAddress delegators must be the same as to the signers
+func CheckSignerAddress(signers, delegators []sdk.AccAddress) bool {
```

Comment:
> I think CheckSignerAddress should be check only one signer. because evmTx must be only one signer,this func need not check multi signer. And this func can not check []signer{a,b}，[]signer{b,a}

ok